.github/workflows/test.yaml

```diff
@@ -920,7 +923,7 @@ jobs:
         clientEngine: ['library', 'binary']
         os: [ubuntu-20.04] #, windows-latest, macos-latest]
     runs-on: ${{ matrix.os }}
-    concurrency: platforms-${{ matrix.platform }}-${{ matrix.clientEngine }}
+    concurrency: ${{ github.job }}-${{ matrix.platform }}-${{ matrix.clientEngine }}
```

Comment:
This was a potential source for making mistakes. When `concurrency` is just job specific, let's use the job name automatically instead of string.